packages/framework/src/hooks/useScreenContext.tsx

```diff
@@ -80,6 +81,7 @@ const HydrateAtoms: React.FC<
       [appAtom, appContext],
       [themeAtom, themeScope.theme],
       [userAtom, appContext.user],
+      [screenStorageAtom, appContext.storage],
```

Comment:
How does it behave without this? This technically will only be the storage value when the apps is initialized right? Not the latest storage value?

Reply:
Yes, and we need this because of we need get and set method on app `initialized` to handle the onLoad actions and widgets text renders. if storage is accessed through `get` method.

Reply:
This needs to be the state of the storage variable when this screen is rendered, not when the application was loaded.